src/packagedcode/npm.py

```diff
@@ -339,22 +407,20 @@ def split_scoped_package_name(name):
     return ns, name
 
 
-def get_licensing(license_object):
+def get_declared_licenses(license_object):
     """
-    Return actual declared licensing info from a license_data object that can be
-    a string, list, dict or dict list and come from the license or licenses
-    package.json tag.
+    Return the declared licenses, which is a list of order dictionary of licenses
```

Comment:
Is this really true `... which is a list of order dictionary of licenses`?
This need to be updated with the changes below. Effectively this `Return a list of declared license items that can be strings or mappings`